pkg/cluster_agent/kubernetes_event.py

```diff
@@ -121,7 +122,7 @@ def from_watch_dict(cls, raw_data):
             if field not in raw_data:
                 raise InvalidWatchEventException(f"Missing `{field}` in event")
 
-        obj = raw_data["raw_object"]
+        obj = raw_data[cls.OBJECT_FIELD_KEY].to_dict()
```

Comment:
its the equal to the raw api response :) I changed it to make sure we have the original resource version value & type (as it not necessary a numeric value but it might be. lets talk about it)